generators/app/templates/src/redux/middlewares/analyticsMiddleware.js

```diff
@@ -0,0 +1,38 @@
+import Config from 'react-native-config';
+import { NavigationActions } from 'react-navigation';
+import { GoogleAnalyticsTracker } from 'react-native-google-analytics-bridge';
+import { GoogleAnalytics } from 'redux-beacon/targets/react-native';
+import { createMiddleware } from 'redux-beacon';
+
+import { getCurrentRouteName } from '../../utils/navUtils';
+
+// TODO: Declare ANALYTICS_TRACKING_ID in .env
+const target = GoogleAnalytics(Config.ANALYTICS_TRACKING_ID, GoogleAnalyticsTracker);
```

Comment:
I'd remove the TODO comment and throw a warning if the env variable is not defined